apps/web/screens/Category/Category.tsx

```diff
@@ -448,6 +448,14 @@ Category.getInitialProps = async ({ apolloClient, locale, query }) => {
       .then((res) => JSON.parse(res.data.getNamespace.fields)),
   ])
 
+  const categoryExists = !!getArticleCategories.find(
```

Comment:
You could use `.some()` instead, so you don't have to double bang

Reply:
Ty, I made some changes